scripts/includes/common.sh

```diff
@@ -185,11 +185,9 @@ configure_st2_cli_config() {
     sudo mkdir -p ${ROOT_USER_CLI_CONFIG_DIRECTORY}
   fi
 
-  sudo sh -c "cat <<EOT > ${ROOT_USER_CLI_CONFIG_PATH}
-[credentials]
-username = ${USERNAME}
-password = ${PASSWORD}
-EOT"
+  sudo sh -c st2 login --config ${ROOT_USER_CLI_CONFIG_PATH} \
```

Comment:
Any reason why do we need explicit `ROOT_USER_CLI_CONFIG_PATH`?

With that, the logic defined in `configure_st2_cli_config()` needs a cleanup. All dir/config file creation should be a `st2 login -w` responsibility.

Reply:
> Do we really need explicit `ROOT_USER_CLI_CONFIG_PATH`?

Technically, no, but it makes it easier to verify that the two calls to `st2 login` (one for root, one for the user) are doing similar things.

I have found it easier to make more variables in Bash than to hard code everything.

> All dir/config file creation should be a `st2 login -w` responsibility.

Yep, fixed.

Reply:
Thanks for explaining that.

To ensure st2 login config is created for both `user` and `root`, would it be enough to run
1) `st2 login -w` (for user)
2) `sudo st2 login -w` (for root)

Can st2 login handle that correctly?
---
Additionally, there is no such `--config` option for `st2 login` command.
We shouldn't hardcode the dirs in every installer. It's a st2 responsibility to decide on config dirs.

Reply:
> Can st2 login handle that correctly?

Should be able to, yes.

> Additionally, there is no such --config option for st2 login command.

Good catch, thanks! Fixed.

> We shouldn't hardcode the dirs in every installer. It's a st2 responsibility to decide on config dirs.

That's true, but I'm just trying to maintain backwards compatibility. We can take care of that in a different PR.